app/src/main/java/com/inveitix/android/clue/ui/MuseumDetailsActivity.java

```diff
@@ -144,23 +141,14 @@ private void startMapActivity(int museumId) {
         overridePendingTransition(R.anim.pull_in_right, R.anim.push_out_left);
     }
 
-    /**
-     * Change the icon of the FAB according to the map status
-     * @param museum
-     */
-    private void loadDwnBtn(Museum museum) {
+    private void loadDownloadBtn(Museum museum) {
```

Comment:
This documentation is useful. Why the name of the method and description are different?
/**
     * Change the icon of the FAB according to the map status
     * @param museum
     */